src/components/plan/PlanMergeReview.test.ts

```diff
@@ -13,7 +13,13 @@ import type {
 import { ADMIN_ROLE } from '../../utilities/permissions';
 import PlanMergeReview from './PlanMergeReview.svelte';
 
-vi.mock('$env/dynamic/public', () => import.meta.env); // https://github.com/sveltejs/kit/issues/8180
+vi.mock('$env/dynamic/public', () => {
+  return {
+    env: {
+      PUBLIC_COMMAND_EXPANSION_MODE: 'typescript',
+    },
+  };
+}); // https://github.com/sveltejs/kit/issues/8180
```

Comment:
I lost track of what changes happened when - are these mocks still necessary after we separated sequence templates into their own suite?

Reply:
Because this test is a unit test, it's using vitest as the test runner, instead of playwright. This allows the tests to affect the UI a little more dynamically over the course of the test via these mocks, whereas playwright requires the UI to be built prior to running the e2e test.